package.json

```diff
@@ -104,5 +105,11 @@
     "dev": "gulp dev",
     "test-env": "node testing/launch"
   },
+  "browserslist": [
+    "last 2 versions",
+    "ios > 7",
```

Comment:
Why not `ios > 8`?
https://data.apteligent.com/ios/

Reply:
I have seen the separate file `browserslist` in some projects. Is it possible to extract such file here?

Reply:
As for me, not a problem to increase the version of iOS to 9+.
Think that a separate file will be superfluous.

Reply:
@nightskylark 
package.json is easier to read.
.browserslistrc has comments, etc, and it's [not supported automatically](https://github.com/less/less-plugin-autoprefix/issues/31#issuecomment-405000747) by the plugin